### ISSUE TITLE
Usi

### ISSUE THREAD

**Issue by T0nio** [yktoo/indicator-sound-switcher#108]

**Comment by yktoo** [comment]
No, this is not the intended use of this app. This application is essentially a frontend to PulseAudio, and there are lots of other ways to communicate with the daemon. You can do it directly via the API, like SSI does (however that's a bit of a challenge due to the asynchronous nature of PulseAudio), or employ other CLI utilities like `pacmd`.
